src/main/java/seedu/tp/ui/Ui.java

```diff
@@ -326,13 +328,36 @@ public void listAllFlashcardsOrdered(FlashcardList flashcardList) {
         }
     }
 
+    /**
+     * Displays the user's study plan.
```

Comment:
I think the comment can be more specific, e.g. "Displays a list of all the user's study plan(s), sorted by date."

Reply:
Will make the change!